components/shopify_partner/queries/getAppInstalls.js

```diff
@@ -0,0 +1,45 @@
+const { gql } = require("graphql-request");
+
+const query = gql`
```

Comment:
Does it make sense to link to the relevant Shopify GraphQL API docs so users can look up more about the query format?